identity-iota/src/did/url/iota_did.rs

```diff
@@ -130,6 +129,16 @@ impl IotaDID {
     self.0.join(other).map_err(Into::into).and_then(Self::try_from_owned)
   }
 
+  /// Change the method of the [`IotaDID`].
+  pub fn set_method(&mut self, value: impl AsRef<str>) {
```

Comment:
This method can be left out, it should always be `iota`.

Reply:
Done.